### ISSUE TITLE
Data inconsistent with Token City Explorer

### ISSUE THREAD

**Issue by Keith-CY** [Magickbase/ckb-explorer-public-issues#96]
CKB Explorer: https://pudge.explorer.nervos.org/address/ckt1qyqp4ekpdcvd9qrwfpxf8de3ms29mh68302suquspj

Token City Explorer: https://ckb-nft-explorer.staging.token.city/holder/tokens/ckt1qyqp4ekpdcvd9qrwfpxf8de3ms29mh68302suquspj

I've checked that the address received `4240 CKB` of NFTs and sent them all, now it holds 0 cells so no NFTs on its page is reasonable.

But there're many NFTs on the Token City Explorer, are they all unclaimed CoTA?

Maybe @shaojunda could help

**Comment by Keith-CY** [comment]
![image](https://user-images.githubusercontent.com/7271329/198544379-d212871e-3c5c-4f61-a3cf-a0511603450b.png)
![image](https://user-images.githubusercontent.com/7271329/198544412-9e7099e3-43d1-42c3-b1dc-8db633808c9b.png)
![image](https://user-images.githubusercontent.com/7271329/198544729-7565095d-64fc-4efa-8fad-e0cac08ee6a7.png)
![image](https://user-images.githubusercontent.com/7271329/198544755-1905d51d-ff91-465f-a87e-ab0efb2e9784.png)


Seems that #5562 has been sent to a new address by tx https://pudge.explorer.nervos.org/transaction/0x1ccd4f44752cc8f83ce8cbb8e4bee7b0add244721854d065f8f41659f8b4c997

**Comment by shaojunda** [comment]
> CKB Explorer: [pudge.explorer.nervos.org/address/ckt1qyqp4ekpdcvd9qrwfpxf8de3ms29mh68302suquspj](https://pudge.explorer.nervos.org/address/ckt1qyqp4ekpdcvd9qrwfpxf8de3ms29mh68302suquspj)
> 
> Token City Explorer: [ckb-nft-explorer.staging.token.city/holder/tokens/ckt1qyqp4ekpdcvd9qrwfpxf8de3ms29mh68302suquspj](https://ckb-nft-explorer.staging.token.city/holder/tokens/ckt1qyqp4ekpdcvd9qrwfpxf8de3ms29mh68302suquspj)
> 
> I've checked that the address received `4240 CKB` of NFTs and sent them all, now it holds 0 cells so no NFTs on its page is reasonable.
> 
> But there're many NFTs on the Token City Explorer, are they all unclaimed CoTA?
> 
> Maybe @shaojunda could help

yes, you are right.  listed here are all unclaimed CoTA.  It can also be judged by the type of transaction. There is only distribution, no transfer.

**Comment by Keith-CY** [comment]
> > CKB Explorer: [pudge.explorer.nervos.org/address/ckt1qyqp4ekpdcvd9qrwfpxf8de3ms29mh68302suquspj](https://pudge.explorer.nervos.org/address/ckt1qyqp4ekpdcvd9qrwfpxf8de3ms29mh68302suquspj)
> > Token City Explorer: [ckb-nft-explorer.staging.token.city/holder/tokens/ckt1qyqp4ekpdcvd9qrwfpxf8de3ms29mh68302suquspj](https://ckb-nft-explorer.staging.token.city/holder/tokens/ckt1qyqp4ekpdcvd9qrwfpxf8de3ms29mh68302suquspj)
> > I've checked that the address received `4240 CKB` of NFTs and sent them all, now it holds 0 cells so no NFTs on its page is reasonable.
> > But there're many NFTs on the Token City Explorer, are they all unclaimed CoTA?
> > Maybe @shaojunda could help
> 
> yes, you are right. listed here are all unclaimed CoTA. It can also be judged by the type of transaction. There is only distribution, no transfer.

However, in the second [comment](https://github.com/Magickbase/ckb-explorer-public-issues/issues/96#issuecomment-1294710525) I find the NFT is recognized as an m-NFT, based on the backend `cell type` field and the `type script` recorded in the front-end, and the `134 capacity` which is a characteristic of m-NFT.

And from data of the CKB Explorer, a token (#5562 in the comment) listed in https://ckb-nft-explorer.staging.token.city/holder/tokens/ckt1qyqp4ekpdcvd9qrwfpxf8de3ms29mh68302suquspj has been transferred to `ckt1qslpadldfqym94sx2zlfdfq2h776lvlmjs4hkdlvwuy7vn3v6zncxyz0yns47nd6weychvjkty7udk2cwmvgz9klrka`, the receiver holds it now but it is not listed on https://explorer.token.city/holder/tokens/ckt1qqlpadldfqym94sx2zlfdfq2h776lvlmjs4hkdlvwuy7vn3v6zncxqgsfujwzh6dhfmynzaj2evnm3ketpmd3qg2wmvc6

If data on CKB Explorer are correct, those tokens have been sent out so `[ckt1qyqp4ekpdcvd9qrwfpxf8de3ms29mh68302suquspj](https://pudge.explorer.nervos.org/address/ckt1qyqp4ekpdcvd9qrwfpxf8de3ms29mh68302suquspj)` holds nothing, but Token City Explorer didn't detect the transaction
